code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -253,6 +269,22 @@
 		dat += "[surplus] left.<br>"
 	return dat
 
+/datum/spellbook_entry/item/bookofdarkness
+	name = "Book of Darkness"
+	desc = "A forbidden tome, previously outlawed from the Wizard Federation for containing necromancy that is now being redistributed. Contains a powerful artifact that gets stronger with every soul it claims, a stunning spell that deals heavy damage to a single target, an incorporeal move spell and a spell that lets you explode corpses. Comes with a cool set of powerful robes as well that can carry the Staff of Revenant."
+	item_path = /obj/item/weapon/bookofdarkness
+	log_name = "BoD"
+	category = "Assistance"
+	cost = 5
+	limit = 1
+
+/datum/spellbook_entry/item/staffofrevenant
```

Comment:
It defaults to 2 (as intended) and no limit is intended either, in case you want to teach your apprentice how to be villia-necromancers.